### PR TITLE
Use a configuration file to render the available wallets

### PR DESCRIPTION
**Pull request by ronaldsg20**
I've updated the selectBitcoinWalletcomponent using a configuration file to load
		the required data related to each wallet

### CODE REVIEW

**Review by lserra-iov — APPROVED** [review]

**Review by alexjavabraz — APPROVED** [review]
LGTM

File: src/pegin/components/create/ConfirmTx.vue [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+                </span>
+              </v-row>
+              <v-row v-if="step.outputsToshow.opReturn.amount || step.outputsToshow.opReturn.value"
+                 class="mt-2 d-flex justify-center">
+                <span>
+                  OP_RETURN
+                </span>
+                <v-tooltip right>
+                  <template v-slot:activator="{props}">
+                      <v-icon v-bind="props" v-on="props.on" color="grey-lighten-1">
```
Comment (lserra-iov) [inline src/pegin/components/create/ConfirmTx.vue:51]: The icon needs to be imported from the `@mdi/js` package to be properly displayed.

**Review by lserra-iov — DISMISSED** [review]

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Review by alexjavabraz — APPROVED** [review]
LGTM

**Review by lserra-iov — APPROVED** [review]

**Pull request merged**